dt-posts/posts.php

```diff
@@ -310,19 +310,19 @@ public static function format_connection_message( $p2p_id, $activity, $action =
             }
         } else if ( $p2p_type === "groups_to_groups"){
             if ($action == "connected to"){
-                $object_note_to = sprintf( esc_html_x( '%s added to child groups', 'group2 added to child groups', 'disciple_tools' ), $from_title );
+                $object_note_to = sprintf( esc_html_x( '%s added to child groups', 'group1 added to child groups', 'disciple_tools' ), $from_title );
                 $object_note_from = sprintf( esc_html_x( '%s added to parent groups', 'group1 added to parent groups', 'disciple_tools' ), $to_title );
             } else {
-                $object_note_to = sprintf( esc_html_x( '%s removed from child groups', 'group2 removed from child groups', 'disciple_tools' ), $from_title );
+                $object_note_to = sprintf( esc_html_x( '%s removed from child groups', 'group1 removed from child groups', 'disciple_tools' ), $from_title );
                 $object_note_from = sprintf( esc_html_x( '%s removed from parent groups', 'group1 removed from parent groups', 'disciple_tools' ), $to_title );
             }
         }
         else if ( $p2p_type === "groups_to_peers"){
             if ($action == "connected to"){
-                $object_note_to = sprintf( esc_html_x( '%s added to peer groups', 'group2 added to peer groups', 'disciple_tools' ), $from_title );
+                $object_note_to = sprintf( esc_html_x( '%s added to peer groups', 'group1 added to peer groups', 'disciple_tools' ), $from_title );
```

Comment:
Change not needed.

Reply:
I had done it for consistence but so that it doesn't trigger a re translation, I reverted it (as suggested). 
But there was one of these that was duplicated because the context wasn't the same but should have been,

Reply:
sting on line 323 is the same but had a different context so was duplicating the string.

Reply:
peer groups change good